internal/engine/docker_compose_build_and_deployer.go

```diff
@@ -93,12 +94,12 @@ func (bd *DockerComposeBuildAndDeployer) BuildAndDeploy(ctx context.Context, st
 			// service at once, we'll have to match up image build results to DC target by ref.
 			ref, err = bd.icb.Build(ctx, iTarget, currentState[iTarget.ID()], ps, true)
 			if err != nil {
-				return store.BuildResultSet{}, err
+				return store.BuildResultSet{}, errors.Wrap(err, "DockerComposeBuildAndDeployer")
```

Comment:
this seems like a weird thing to show if the user has a compiler error. i'd expect them to ignore it and blame us, which is not what we want.

Reply:
Ah but it already tags them with "ImageBuild:", which to me is misleading about the specific deployer we're using. though now that i think of it, that's probably way more confusing to us than to a user. i'll take this out.